### ISSUE TITLE
'in' in IN() is INVALID

### ISSUE THREAD

**Issue by trel** [irods/irods#3886]
### iRODS Version, OS and Version
4.2.2

### What did you try to do?
Run iquest with the string 'in' within an 'IN()' clause.

### Expected behavior
'in' should not trip up the GenQuery "parser"

### Observed behavior (including steps to reproduce, if applicable)

This returns what is expected... no results
```
$ iquest "SELECT DATA_NAME WHERE DATA_NAME IN ('wi')"
CAT_NO_ROWS_FOUND: Nothing was found matching your query
```

This thinks I've passed bad syntax to GenQuery
```
$ iquest "SELECT DATA_NAME WHERE DATA_NAME IN ('win')"
 ERROR: iquest Error: queryAndShowStrCond failed status = -816000 CAT_INVALID_ARGUMENT
```

**Comment by trel** [comment]
duplicate of https://github.com/irods/irods/issues/3886  ?

**Comment by trel** [comment]
duplicate of https://github.com/irods/irods/issues/3064 ?

**Comment by trel** [comment]
This will be fixed by #3902

**Comment by korydraughn** [comment]
> duplicate of #3064 ?

#3064 has been resolved. See that issue for the commits.

Closing.
